python/raf/frontend/pytorch.py

```diff
@@ -86,6 +86,12 @@ def inner(model, shape_dict):
             if input_type.startswith("int64"):
                 max_val = 10000 if len(input_info) == 2 else input_info[2]
                 input_data = torch.randint(max_val + 1, input_shape, device=device)
+            # when the dtype is bf16/fp16, we make up a float32 input to do the
+            # tracing in cpu with Ratex
```

Comment:
This frontend is not necessary to be tightly-coupled with Ratex.

Reply:
I have made up the input dtype in the Ratex side. Then I will close this PR. Thanks!